processor/cumulativetodeltaprocessor/processor.go

```diff
@@ -15,19 +17,23 @@ import (
 )
 
 type cumulativeToDeltaProcessor struct {
-	includeFS       filterset.FilterSet
-	excludeFS       filterset.FilterSet
-	logger          *zap.Logger
-	deltaCalculator *tracking.MetricTracker
-	cancelFunc      context.CancelFunc
+	includeFS          filterset.FilterSet
+	excludeFS          filterset.FilterSet
+	includeMetricTypes []string
+	excludeMetricTypes []string
```

Comment:
Lets make these new types maps.  As part of the `newCumulativeToDeltaProcessor` we can convert the configured metric type list to a map of metric type enums. That will make the hot-path check faster.